src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -224,8 +224,9 @@ MUST be present **in this specific order**:
 
 {{ MACROS___make_columns_table(
    {
-      "name__channels": ("REQUIRED", "The label must correspond to `_electrodes.tsv` name "
-                         "and all ieeg type channels are required to have a position."),
+      "name__channels": ("REQUIRED", "When a corresponding electrode is specified in `_electrodes.tsv`, "
+                         "the name of that electrode MAY be specified here and the reference electrode "
+                         "name MAY be provided in the `reference` column of the `_electrodes.tsv` file."),
```

Comment:
the `reference` column is not part of the `_electrodes.tsv` file but of this `_channels.tsv` file.